code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -322,7 +323,7 @@
 	item_state = "xenos_helm"
 	body_parts_covered = CHEST|GROIN|LEGS|ARMS|HANDS
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
-
+	allowed = list(/obj/item/clothing/mask/facehugger/toy)

```

Comment:
Well, you can take them off your face if you recover from the stun before they finish tonguing you.